kge/model/kge_model.py

```diff
@@ -317,8 +383,48 @@ def __init__(
                 dataset,
                 self.configuration_key + ".relation_embedder",
                 num_relations,
+                init_for_load_only=init_for_load_only,
             )
 
+            if not init_for_load_only:
+                # load pretrained embeddings
+                pretrained_entities_filename = self.get_option(
+                    "entity_embedder.pretrain.model_filename"
+                )
+                pretrained_relations_filename = self.get_option(
+                    "relation_embedder.pretrain.model_filename"
+                )
+
+                def load_pretrained_model(
+                    pretrained_filename: str,
+                ) -> Optional[KgeModel]:
+                    if pretrained_filename != "":
+                        self.config.log(
+                            f"Initializing with embeddings stored in "
+                            f"{pretrained_filename}"
+                        )
+                        checkpoint = load_checkpoint(pretrained_filename)
+                        return KgeModel.create_from(checkpoint)
+                    return None
+
+                pretrained_entities_model = load_pretrained_model(
+                    pretrained_entities_filename
+                )
+                if pretrained_entities_filename == pretrained_relations_filename:
+                    pretrained_relations_model = pretrained_entities_model
+                else:
+                    pretrained_relations_model = load_pretrained_model(
+                        pretrained_relations_filename
+                    )
+                if pretrained_entities_model is not None:
```

Comment:
use the functions to get the embedders. also, make sure that subject and object embedders are identical (same object), otherwise raise an error. We can support initializing subject/object embeddings separately later on, if we actually need it